docs/development/_index.md

```diff
@@ -170,7 +170,7 @@ spec:
         - -html-static-dir
         - /headlamp/frontend
         - -in-cluster
-        - -plugins-dir=/headlamp/plugins
+        - -plugins-dir=/headlamp/.plugins
```

Comment:
This is a breaking change for any deployments who just update the image tag, whereas before I said we could break just with how it was generated, having .plugins in the repo file. I am happy to clarify a bit better tomorrow.